### DISCUSSION TITLE
Semi-hide tab bar

### DISCUSSION THREAD

**Discussion by Oscillope** [qutebrowser/qutebrowser#6288] [Ideas] [answered]
I was having a conversation with some coworkers about vertical tab bars and had an idea that would make vertical tabs nicer, for me personally at least. Basically, what I'm looking for is to have the tab bar collapsed to only show the tab number and favicon most of the time, and then expand (to `tabs.width`) to show the tab title on either mouseover or any tab-manipulating keyboard events. Would be nice to have some idea of which tabs are open (as opposed to `tabs.show = switching`) without permanently occupying that much horizontal space.

**Comment by The-Compiler** [comment, marked as answer]
Sorry, this got lost for a bit! I added a comment to #5480 which is somewhat similar.

**Reply by Oscillope** [reply]
No worries, thanks for the response! And for making such a great browser :)
